packages/terra-alert/translations/es-ES.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
These aren't spanish?

Reply:
they are placeholders for translation team to update. Need to log a jira in trans team so they can update the json and send a PR

Reply:
I'm ok with landing the PR with english translations as placeholders for the new locales. Its not ideal, but it avoids teams getting JS error is they use these new locales. HL team is on a time crunch to add this to one of their projects. I'll work with the translation team to prioritize getting these translations asap so we don't go too long with placeholder translations.